base/workflow/R/run.write.configs.R

```diff
@@ -27,10 +26,14 @@
 #'
 #' @author David LeBauer, Shawn Serbin, Ryan Kelly, Mike Dietze, Akash B V
 
-run.write.configs <- function(settings, ensemble.size, write = TRUE,
+run.write.configs <- function(settings, ensemble.size, input_design, write = TRUE,
                               posterior.files = rep(NA, length(settings$pfts)),
-                              overwrite = TRUE, input_design = NULL,
-                              input_design_ens = NULL, input_design_sa = NULL) {
+                              overwrite = TRUE) {
+
+  # extract designs from input_design list
+  input_design_ens <- if (!is.null(input_design)) input_design$ensemble else NULL
+  input_design_sa <- if (!is.null(input_design)) input_design$sensitivity else NULL
```

Comment:
Per earlier comments, I'd recommend only having one input_design, not separate ens and sa designs